examples/api/lib/material/selectable_region/selectable_region.0.dart

```diff
@@ -72,14 +72,16 @@ class _SelectableAdapter extends SingleChildRenderObjectWidget {
   }
 
   @override
-  void updateRenderObject(BuildContext context, _RenderSelectableAdapter renderObject) {
+  void updateRenderObject(
```

Comment:
I think we should just keep it one line for readability

Reply:
can you revert the format change?

Reply:
What I ended up doing since my Visual Studio Code setup auto formats every time I save, so it'd be pretty hard to unformat all the files I touched manually. My initial run of dart format was reversed.

What I ended up doing was just reformatted that folder with dart format with a bigger line length. Let me know if that works for you.

Reply:
No there are still a bunch of formatting change. the framework repo does not have format turn on, I would suggest to start from scratch and manually apply the actual change.

Reply:
Alright; I will do that then